src/get.cpp

```diff
@@ -1,73 +1,58 @@
 #include "r_trie.h"
 
+template <typename T>
+static inline std::vector < std::string > get_keys_generic(SEXP radix){
+  r_trie <T>* rt_ptr = (r_trie <T> *) R_ExternalPtrAddr(radix);
+  ptr_check(rt_ptr);
```

Comment:
Oh clever, I thought about doing this but was so tired I got the wrong end of the stick and concluded it wouldn't work.